main/Accordion/AccordionItem.tsx

```diff
@@ -130,49 +117,52 @@ const AccordionItem: FC<Props> = (props) => {
         ],
       }}
     >
-      {element || <StyledIcon theme={theme} name="chevron-down-light" />}
+      {element}
     </Animated.View>
   );
 
   useEffect(() => {
-    if (bodyMounted) {
-      dropDownAnimValueList.setValue(collapsed ? -bodyHeight : 0);
-    }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-  }, [bodyMounted]);
-
-  useEffect(() => {
-    const targetValue = collapsed ? -bodyHeight : 0;
+    const targetValue = collapsed ? 0 : 1;
 
     if (!shouldAnimate) {
-      dropDownAnimValueList.setValue(targetValue);
+      rotateAnimValue.setValue(targetValue);
     }
 
-    Animated.timing(dropDownAnimValueList, {
+    Animated.timing(rotateAnimValue, {
       toValue: targetValue,
-      duration: animDuration || 300,
-      useNativeDriver: true,
+      duration: 200,
+      easing: Easing.linear,
+      useNativeDriver: false,
     }).start();
 
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [collapsed]);
 
+  const targetValueForDropdown = useMemo(() => {
+    return collapsed ? 0 : bodyHeight;
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [collapsed]);
+
   useEffect(() => {
-    const targetValue = collapsed ? 0 : 1;
+    if (bodyMounted) {
+      dropDownAnimValue.setValue(targetValueForDropdown);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [bodyMounted]);
 
```

Comment:
Please remove `eslint-disable-next-line react-hooks/exhaustive-deps` and fix the code.
You may refer to https://kyounghwan01.github.io/blog/React/exhaustive-deps-warning/

Reply:
I fixed code to remove `eslint-disable-next-line react-hooks/exhaustive-deps`
1. I use `useRef` to remove effect dependencies to animation value and bodyHeight.
2. Animated controls are merged with one action.
3. I add a `startAnimation` function to effect dependencies but wrap its definition into the `useCallback` Hook.